.github/workflows/create-prs-to-update-vcs-repositories.yaml

```diff
@@ -29,3 +29,15 @@ jobs:
           new_branch_prefix: feat/update-
           autoware_repos_file_name: autoware.repos
           verbosity: 0
+
+      - name: Create PRs to update VCS repositories for simulator.repos
+        uses: autowarefoundation/autoware-github-actions/create-prs-to-update-vcs-repositories@v1
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          repo_name: autowarefoundation/autoware
+          parent_dir: .
+          targets: major minor patch
+          base_branch: main
+          new_branch_prefix: feat/update-
+          autoware_repos_file_name: simulator.repos
```

Comment:
It will also be used for purposes other than `autoware.repos`, so `repos_file` or `repos_file_name` instead of `autoware_repos_file_name` are better. Please make the modification PR next.